elements/pfe-accordion/src/pfe-accordion.scss

```diff
@@ -3,18 +3,28 @@
 $LOCAL: accordion;
 
 :host {
+  --pfe-accordion--main:         #{pfe-color(surface--lightest)};
+  --pfe-accordion--aux:          #{pfe-color(surface--lightest--text)};
+
+  --pfe-accordion--accent:       transparent;
+
   display: block;
   position: relative;
   overflow: hidden;
   margin: 0;
-  color: #{pfe-radio(color--text)};
 }
 
-:host([on="dark"]) {
-  @include pfe-theme($theme: "dark");
+// Only invoke the light broadcast settings if on="dark" is not present
+:host(:not([on="dark"])) {
+  @include pfe-set-broadcast(lightest);
 }
 
-:host([on="light"]) {
-  @include pfe-theme($theme: "light");
-}
 
+/// ===========================================================================
+/// CONTEXT STYLES, ON=DARK
+/// ===========================================================================
+
+:host([on="dark"]) {
```

Comment:
@starryeyez024 I can't think of an easy way to do away with on="dark" on the accordions yet so we'll have to keep this one for now.  Let's keep noodling on if there's a way to influence it based on broadcast context but for now, this seems to get us pretty close.